common/src/world.rs

```diff
@@ -19,10 +19,14 @@ pub enum Material {
     Graveldirt = 15,
     Blackstone = 16,
     Bigflowergrass = 17,
+    GreyBrick = 18,
+    WoodPlanks = 19,
+    WhiteBrick = 20,
+    ReservedVoid = 21,
```

Comment:
This doesn't seem to be used, and represents an approach I'd rather avoid. We shouldn't use voxel data to encode internal details of the terrain generation algorithm; it's only for the final output of the algorithm.